express-middlewares/expressMiddlewares.js

```diff
@@ -1,8 +1,34 @@
+const express = require('express');
 
-function expressMiddlewares(...args) {
-  return args;
-}
+const app = express();
 
-export {
-  expressMiddlewares,
-};
+app.use((req, res, next) => {
+  const url = req.originalUrl.slice(1);
+  res.setHeader('middleware-header', url);
+  console.log(`Some user visited ${url} at ${new Date()}`);
+  next();
+});
+
+app.get('/', (req, res) => {
+  res.send('Hello');
+});
+app.get('/languages', (req, res) => {
+  const languages = ['java', 'js', 'python', 'ruby', 'haskell'];
+  res.setHeader('Content-Type', 'application/json');
+  res.send(languages);
```

Comment:
You could use `res.json()`. I think it does the same thing and might be in better way.
Here is one blog post giving short explanation: https://blog.fullstacktraining.com/res-json-vs-res-send-vs-res-end-in-express/